### DISCUSSION TITLE
Query results do not match the actual database values for repeat queries

### DISCUSSION THREAD

**Discussion by coldenol** [dbeaver/dbeaver#8813] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version: Windows 10
- DBeaver version: 7.0.4.202005031953

#### Connection specification:
- AWS Aurora / MySQL 5.7

#### Describe your question:
I often face this issue while executing big queries. The result values I got from the first query execution are displayed for all next executions despite real data was changed in the database. Only reconnecting solves the problem but not for long.
It looks like I get the result from the cache.

I didn't find such an issue among opened ones. Maybe this is not an issue and I should just enable/disable something in the application settings?

**Comment by serge-rider** [comment, marked as answer]
If you are in manual transaction mode and you did some changes (e.g. updated or added some rows) then you won't see external changes until you end your transaction (perform Commit or Rollback).

Or you can switch to auto-commit or smart commit mode.

**Comment by coldenol** [comment]
You are right! Switching to auto-commit from manual-commit mode solved my issue.
I do not use transactions in IDEs so I did not think of that.
Thank you!
